scripts/release_support/changelog.sh

```diff
@@ -58,8 +58,34 @@ export TZ=$time_zone;ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ
 
 pushd $clone_dir/clone
 
-wget -q $BUILD_LOCATION/$START_BUILD/dev/RELEASE.INFO -O start_build_manifest.txt
-wget -q $BUILD_LOCATION/$TARGET_BUILD/dev/RELEASE.INFO -O target_build_manifest.txt
+if [ -z $BUILD_LOCATION ]; then
+	CHECK_INPUT_VARIABLE_STATUS=$(echo "$START_BUILD"|grep RELEASE.INFO|wc -l)
+	if [ $CHECK_INPUT_VARIABLE_STATUS == "0" ]; then
+		docker pull "$START_BUILD"
+		docker run --rm "$START_BUILD" cat /RELEASE.INFO > start_build_manifest.txt
+		docker pull "$TARGET_BUILD"
+		docker run --rm "$TARGET_BUILD" cat /RELEASE.INFO > target_build_manifest.txt
+		START_BUILD=$(grep "BUILD:" start_build_manifest.txt| awk '{print $2}'|tr -d '"')
+		TARGET_BUILD=$(grep "BUILD:" target_build_manifest.txt| awk '{print $2}'|tr -d '"')
+	else
+
+		wget -q $START_BUILD -O start_build_manifest.txt
+		if [ $? -ne 0 ]; then
+		    echo "ERROR:While downloading start build RELEASE INFO by wget command got failed for $START_BUILD"
+		    exit 1
+		fi
+		START_BUILD=$(echo "$START_BUILD"|awk -F "/" '{print $9}')
+		wget -q $TARGET_BUILD -O target_build_manifest.txt
+		if [ $? -ne 0 ]; then
+		    echo "ERROR:While downloading target build RELEASE INFO by wget command got failed $TARGET_BUILD"
```

Comment:
Update error message as below.
"ERROR: Downloading RELEASE.INFO file got failed for $TARGET_BUILD"

Reply:
Changed error msg